server/app/scanpy_engine/scanpy_engine.py

```diff
@@ -140,6 +141,14 @@ def _create_schema(self):
                     )
                 self.schema["annotations"][ax].append(ann_schema)
 
+        for layout in self.config['layout']:
+            ly_schema = {
```

Comment:
nit: `ly` is not how I would abbreviate layout (`lo` could be defensible, but I'd prefer `layout`)

Reply:
fixed